op-acceptance-tests/tests/interop/seqwindow/expiry_test.go

```diff
@@ -83,13 +83,34 @@ func TestSequencingWindowExpiry(gt *testing.T) {
 		"tx", receipt2.TxHash, "l2Block", old)
 	// The logs will show a "Chain reorg detected" from op-geth.
 
+	// Once this happens we don't want to continue to try and include the tx in a block again,
+	// since that will then be reorged out again.
+	// We need to enter recovery-mode to not continue to build an incompatible chain that will not get submitted.
+	// It may reorg once more, but then stays compatible.
+	// For a while we'll have to build blocks that are not going to be reorged out due to subtle L1 origin divergence.
+	t.Logger().Info("Turning on recovery-mode")
+	t.Require().NoError(sys.L2CLA.Escape().RollupAPI().SetRecoverMode(t.Ctx(), true))
```

Comment:
just for my understanding, what's `Escape()` doing?

Reply:
It's an intentional thing, to make it clear when we go from the DSL world to the full-capabilities world. We can introduce DSL for the API later, once we see a need for it.

Reply:
got it, thanks